[2024-10-03T22:38:49.951-0400] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T22:38:55.743-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_extracted_texts_task manual__2024-10-04T02:24:10.297091+00:00 [queued]>
[2024-10-03T22:38:55.959-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_extracted_texts_task manual__2024-10-04T02:24:10.297091+00:00 [queued]>
[2024-10-03T22:38:55.961-0400] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-10-03T22:38:56.260-0400] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): upload_extracted_texts_task> on 2024-10-04 02:24:10.297091+00:00
[2024-10-03T22:38:56.263-0400] {standard_task_runner.py:72} INFO - Started process 31318 to run task
[2024-10-03T22:38:56.266-0400] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'huggingface_pdf_pipeline', 'upload_extracted_texts_task', 'manual__2024-10-04T02:24:10.297091+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/huggingface_pdf_airflow_dag.py', '--cfg-path', '/tmp/tmpm23f82rb']
[2024-10-03T22:38:56.267-0400] {standard_task_runner.py:105} INFO - Job 37: Subtask upload_extracted_texts_task
[2024-10-03T22:39:11.311-0400] {job.py:229} INFO - Heartbeat recovered after 34.82 seconds
[2024-10-03T22:39:18.254-0400] {task_command.py:467} INFO - Running <TaskInstance: huggingface_pdf_pipeline.upload_extracted_texts_task manual__2024-10-04T02:24:10.297091+00:00 [running]> on host LAPTOP-GD5THE3N.
[2024-10-03T22:39:28.764-0400] {job.py:229} INFO - Heartbeat recovered after 17.54 seconds
[2024-10-03T22:39:31.884-0400] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='huggingface_pdf_pipeline' AIRFLOW_CTX_TASK_ID='upload_extracted_texts_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T02:24:10.297091+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T02:24:10.297091+00:00'
[2024-10-03T22:39:31.886-0400] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-03T22:39:31.888-0400] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-03T22:39:31.890-0400] {logging_mixin.py:190} INFO - Current task name:upload_extracted_texts_task state:running start_date:2024-10-04 02:38:55.745534+00:00
[2024-10-03T22:39:31.891-0400] {logging_mixin.py:190} INFO - Dag name:huggingface_pdf_pipeline and current dag run status:running
[2024-10-03T22:39:31.902-0400] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T22:39:31.903-0400] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/User/Documents/GitHub/DAMG7245/big_data_assignment2/airflow/dags/modules/pipeline.py", line 229, in upload_extracted_texts_to_azure
    for pdf_path, extracted_text in extracted_texts:
ValueError: not enough values to unpack (expected 2, got 1)
[2024-10-03T22:39:35.837-0400] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-10-03T22:39:35.838-0400] {logging_mixin.py:190} INFO - Task start:2024-10-04 02:38:55.745534+00:00 end:2024-10-04 02:39:35.836746+00:00 duration:40.091212
[2024-10-03T22:39:35.840-0400] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): upload_extracted_texts_task> dag:<DAG: huggingface_pdf_pipeline> dagrun:<DagRun huggingface_pdf_pipeline @ 2024-10-04 02:24:10.297091+00:00: manual__2024-10-04T02:24:10.297091+00:00, state:running, queued_at: 2024-10-04 02:24:11.104549+00:00. externally triggered: True>
[2024-10-03T22:39:35.840-0400] {logging_mixin.py:190} INFO - Failure caused by not enough values to unpack (expected 2, got 1)
[2024-10-03T22:39:35.841-0400] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=huggingface_pdf_pipeline, task_id=upload_extracted_texts_task, run_id=manual__2024-10-04T02:24:10.297091+00:00, execution_date=20241004T022410, start_date=20241004T023855, end_date=20241004T023935
[2024-10-03T22:39:36.217-0400] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T22:39:36.218-0400] {standard_task_runner.py:124} ERROR - Failed to execute job 37 for task upload_extracted_texts_task (not enough values to unpack (expected 2, got 1); 31318)
Traceback (most recent call last):
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/patrick/.cache/pypoetry/virtualenvs/airflow-wPbgcfzP-py3.10/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/User/Documents/GitHub/DAMG7245/big_data_assignment2/airflow/dags/modules/pipeline.py", line 229, in upload_extracted_texts_to_azure
    for pdf_path, extracted_text in extracted_texts:
ValueError: not enough values to unpack (expected 2, got 1)
[2024-10-03T22:39:36.251-0400] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-03T22:39:36.310-0400] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-10-03T22:39:39.406-0400] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T22:39:39.453-0400] {local_task_job_runner.py:245} INFO - ::endgroup::
